dev_tools/notebooks/notebook_test.py

```diff
@@ -40,7 +40,8 @@
     # chemistry.ipynb requires openfermion, that installs cirq 0.9.1, which interferes
     # with testing cirq itself...
     'docs/tutorials/educators/chemistry.ipynb',
-    'docs/noise.ipynb',
+    'docs/noise/qcvv/xeb_calibration_example.ipynb',
```

Comment:
could you add a comment on the reasons for skipping these?

Reply:
docs/noise no longer exists: is now noisy_simulation.ipynb and doesn't use auth. 

docs/noise/qcvv/xeb_calibraiton_example.ipynb uses auth in the third code cell and was causing CI issues because (I think) the output of that cell was cleared. It wasn't getting skipped even though the other cells have outputs. 
See: the error for the commit before ["Add notebook skip exceptions to notebook_test"](https://github.com/quantumlib/Cirq/pull/5442/commits/0a7a4d8436a65345f20925eac024b3d9bd562505): 
https://github.com/quantumlib/Cirq/runs/6731074286?check_suite_focus=true

Reply:
I think Dave meant to add a comment in the notebook_test.py file so that later developers know why it is currently being skipped.

Reply:
Added.